README.md

```diff
@@ -1,2 +1,39 @@
+
 # Hello-Microverse
-hello microverse project learning 
+
```

Comment:
- [ ] You have done a good job working on the README.md file, However for it to be even more professional it could be better to give it a more descriptive description and also give some instructions on how a non-technical person can set up the project locally in the getting started section, This will make your Repository more professional 👍 .
Let's work on this issue by following the given template you can find [here](https://github.com/microverseinc/readme-template).